webpack/app/components/temporaryidentifierwidget.coffee

```diff
@@ -208,19 +233,27 @@ class TemporaryIdentifierWidgetController
 
     # Fields to include on search results
     fields = [
-      "PatientFullName"
-      "PatientAddress"
-      "DateOfBirth"
-      "Age"
-      "Sex"
+      "Title"
+      "name"
+      "surname"
+      "age"
+      "birthdate"
+      "gender"
+      "email"
+      "address"
+      "zipcode"
+      "city"
+      "country"
     ]
 
     deferred = $.Deferred()
     options =
       url: @get_portal_url() + "/@@API/read"
       data:
-        catalog_name: "bika_catalog_analysisrequest_listing"
-        medical_record_number: mrn
+        portal_type: "Patient"
+        catalog_name: "portal_catalog"
+        patient_mrn: mrn
+        is_active: yes  # search only active patients
```

Comment:
We have to search for "inactive" patients too. And if the patient is in such status, call user to action:
- Please, re-activate the patient
- Use another MRN